doc/rados/configuration/pool-pg-config-ref.rst

```diff
@@ -66,6 +66,13 @@ Ceph configuration file.
 :Default: ``1``. Typically a host containing one or more Ceph OSD Daemons.
 
 
+``osd crush initial weight``
+:Description: The initial crush weight for the new osd just added into crushmap.
+:Type: Double
+:Default: ``0``. The initial crush weight for the new osd added into crushmap, by
```

Comment:
this is not accurate, the default value is calculated using the size of the osd data volume . it's something like "sizeof($volume)/1TB".